hammers/scripts/clean_old_aggregates.py

```diff
@@ -63,21 +77,75 @@ def clear_aggregates(agg_list):
 
     return errors, report
 
+def orphan_find(allaggs):
+
+    # Find all hosts currently in aggregates
+    hosts_from_aggs = []
+    for agg in allaggs.values():
+        for host in agg['hosts']:
+            hosts_from_aggs.append(host)
+
+    # Make list of ironic hosts not in any aggregate
+    ironic_hosts = osrest.ironic.nodes(auth, details=False).keys()
+    orphans = []
+    for host in ironic_hosts:
+        if host not in hosts_from_aggs:
+            host_id = [h['id'] for h in osrest.blazar.hosts(auth).values() if h['uid'] == host][0]
+            orphans.append(host_id)
+    return(orphans)
```

Comment:
This style isn't used/recommended 

```suggestion
    return orphans
```